utility/enable-bvls-hmcts-link-guest-pin.sh

```diff
@@ -51,4 +51,4 @@ kubectl -n "$NAMESPACE" rollout restart deployments/hmpps-book-a-video-link-ui
 
 echo
 echo "Restarting Daily Schedule deployment on namespace $NAMESPACE"
-kubectl -n "$NAMESPACE" rollout restart deployments/hmpps-video-conference-schedule-ui
+kubectl -n hmpps-book-a-video-link-dev rollout restart deployments/hmpps-video-conference-schedule-ui
```

Comment:
Should not be committed? Hardcoded namespace?

Reply:
Good spot.  Done in error.